ONSdigital/eq-author-app
# Ear 1360 drag and drop mvp

### What is the context of this PR?

This PR introduces Drag and Drop into the secondary navigation menu, following the same style as. the P.O.C in an earlier ticket.

### How to review

Try it out! Try dragging and dropping under different scenarios, and see what happens.

You shouldn't be able to:

- Drag an item from a folder into the parent section, and vice versa.
- Drag a folder inside a folder.
- Drag anything between folders.
- Drag anything between sections.

You should be able to:

- Drag a page and confirmation question together.
- Reorder items in a section.
- Reorder items in a folder.
- Still use the old move modal to move content.

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process
